src/zope/i18n/__init__.py

```diff
@@ -99,6 +131,29 @@ def translate(msgid, domain=None, mapping=None, context=None,
 
     >>> print(translate(u"eek", 'your.domain'))
     test-from-your.domain
+
+    If no target language is provided, but a context is and we were able to
+    find a translation domain, we will use the `negotiate` function to
+    attempt to determine the language to translate to:
+
+    .. doctest::
+        :hide:
+
+        >>> from zope import i18n
+        >>> old_negatiate = i18n
+
+    >>> def test_negatiate(context):
+    ...     print("Negatiating for %r" % (context,))
```

Comment:
Typo: negatiate -> negotiate (in several places, with different capitalizations).